src/api/middleware/security/enhanced-security.ts

```
@@ -325,7 +325,7 @@ function getObjectDepth(obj: any): number {
   }
 
   if (Array.isArray(obj)) {
-    return 1 + Math.max(0, ...obj.map(getObjectDepth));
+    return 1 + Math.max(0, ...obj?.map(getObjectDepth));
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)